source/guides/stacks/dars.rst

```diff
@@ -41,78 +42,120 @@ in the |CL|-based containers may not be the latest released by |CL|.
 Using the Docker images
 ***********************
 
-#. To immediately start using the latest stable DARS images, pull an image
+Launching the Image
+===================
+
+#. To use the latest stable DARS images, pull an image
    directly from `Docker Hub`_. This example uses the
    `DARS with Intel® MKL`_ Docker image.
 
+   .. code-block:: bash
+
+      docker pull clearlinux/stacks-dars-mkl
+
+
 #. Once you have downloaded the image, you can run it with
 
    .. code-block:: bash
 
       docker run -it --ulimit nofile=1000000:1000000 --name mkl <name of image>
 
-   This will launch the image and drop you into a bash shell inside the
-   container.  You will see output similar to the following:
+   This will launch the image and drop you into a bash shell inside the container. The :command:`--ulimit nofile=` parameter is required in order to increase the allowed number of open files for the Apache Spark engine.
```

Comment:
One question on `docker run` .  For `<name of image>`, do you expect users to run a `docker ps` to find the name of the image first?

Reply:
added -- good question.